### ISSUE TITLE
component's location in preference window

### ISSUE THREAD

**Issue by TotoCoelo** [ghDaYuYu/foo_discogs#2]
If you look at the preferences window. I think the location of the component in the items list should be changed. There is already another item for tagging under `Tools>Tagging` (see the image). Maybe it is a good idea to move it to that location, if possible. 

![bug report](https://user-images.githubusercontent.com/48214068/102603938-cb92fa00-4123-11eb-876a-021ff8e3102d.png)

**Comment by ghDaYuYu** [comment]
You are right, maybe a good idea yes....

Though I do not see the Tagging group nor the freedb Tagger in my fb2k v1.5 installation.
Surely it's been an update on v1.6. I am still using v1.5 too, also during development, to test components backward compatibility. This might be the reason I overlooked this change, thanks for noticing!
I guest I did miss that also because the 'Tag writing' group is offered by the API, and seem the right place to be in.
Need to give it some thought, or even ask to get some confirmation before changing it though...

Thanks for noticing!
Best regards
